distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -199,37 +198,64 @@ When searching with wildcards, do not include the punctuation at the beginning o
 Text searches are by default case insensitive, but case sensitive searches are an option.
 
 [[_temporal_search_details]]Temporal Search Details:: Search based on absolute time of the created, modified, or effective date.
+[no-bullet]
+*** *Any*: Search without any time restrictions (default).
+*** *After*: Search records after a specified time.
+*** *Before*: Search records before a specified time.
+*** *Between*: Set a beginning and end time to search between.
+*** *Relative*: Search records relative to the current time.
 +
-*Any*: Search without any time restrictions (default).
+** [[_spatial_search_details]]*Spatial Search Details*
+[no-bullet]
+*** Search by latitude/longitude, USNG/MGRS, or UTM using a line, polygon, point-radius, or bounding box. Spatial criteria can also be defined by entering a *Keyword* for a region, country, or city in the *Location* section of the query builder.
 +
-*After*: Search records after a specified time.
+** [[_type_search_details]]*Type Search Details*
+[no-bullet]
+*** Search for specific content types.
 +
-*Before*: Search records before a specified time.
+* *Advanced*: Advanced query builder can be used to create more specific searches than can be done through the other methods.
+[no-bullet]
+** [[_advanced_query_builder_details]]*Advanced Query Builder Details*
+[no-bullet]
+*** *Operator*: If 'AND' is used, all the filters in the branch have to be true for this branch to be true. If 'OR' is used, only one of the filters in this branch has to be true for this branch to be true.
+*** *Property*: Property to compare against.
+*** *Comparison*: How to compare the value for this property against the provided value. Depending on the type of property selected, various comparison values will be available. See <<_advanced_query_builder_comparators,Types of Comparators>>
+*** *Search Terms*: The value for the property to use during comparison.
+*** *Sorting*: Sort results by relevance, distance, created time, modified time or effective time.
+*** *Sources*: Perform an enterprise search (the local ${ddf-catalog} and all federated sources) or search specific sources.
 +
-*Between*: Set a beginning and end time to search between.
-+
-*Relative*: Search records relative to the current time.
-
-[[_spatial_search_details]]Spatial Search Details:: Search by latitude/longitude, USNG/MGRS, or UTM using a line, polygon, point-radius, or bounding box. Spatial criteria can also be defined by entering a *Keyword* for a region, country, or city in the *Location* section of the query builder.
-
-[[_type_search_details]]Type Search Details:: Search for specific content types.
-
-* *Advanced*: Advanced query builder to create more specific searches.
+** [[_advanced_query_builder_comparators]]*Advanced Query Builder Comparators*
+[no-bullet]
+*** *Textual*:
+[no-bullet]
+**** *CONTAINS*: Equivalent to <<_text_search_details,Basic Text Search>> with Matchcase set to No.
```

Comment:
I have a suggestion that is outside the scope of the ticket but might be useful. Consider using the in-line code formatting for keywords that would actually appear in the search string. For example,

`CONTAINS`, `NEAR`, and `>=`

instead of 

**CONTAINS**, **NEAR**, and **>=**